torchvision/models/quantization/mobilenetv3.py

```diff
@@ -135,20 +136,32 @@ def _mobilenet_v3_model(
         if "backend" in weights.meta:
             _ovewrite_named_param(kwargs, "backend", weights.meta["backend"])
     backend = kwargs.pop("backend", "qnnpack")
+    quantization_workflow_type = kwargs.pop("quantization_workflow_type", QuantizationWorkflowType.EAGER_MODE)
 
-    model = QuantizableMobileNetV3(inverted_residual_setting, last_channel, block=QuantizableInvertedResidual, **kwargs)
-    _replace_relu(model)
+    if quantization_workflow_type == QuantizationWorkflowType.FX_GRAPH_MODE:
+        model = MobileNetV3(inverted_residual_setting, last_channel, **kwargs)
+    else:
+        model = QuantizableMobileNetV3(inverted_residual_setting, last_channel, block=QuantizableInvertedResidual, **kwargs)
+        _replace_relu(model)
 
     if quantize:
-        model.fuse_model(is_qat=True)
-        model.qconfig = torch.ao.quantization.get_default_qat_qconfig(backend)
-        torch.ao.quantization.prepare_qat(model, inplace=True)
+        # TODO: This shouldn't be QAT?
```

Comment:
As you see MobiletNetV3 doesn't use `quantize_model` but instead does this very weird thing. This was done to avoid an old bug (see https://github.com/pytorch/vision/pull/3323/commits/274c6a1393384054876d701ffa1b54eb6750f1d8#r569264679) in which when we loaded back the results of the QAT and tried to use them with `--test-only` the accuracy was extremely low. But if we instead loaded the QAT unconverted weights, and did the conversion on the fly it worked.

Reply:
I see, that is very weird indeed. Maybe we should add a comment here and link to that issue to provide some context

Reply:
Good call. Added a comment at https://github.com/pytorch/vision/pull/5891 and documented the issue at https://github.com/pytorch/vision/issues/5890